.github/workflows/versioning.yml

```
@@ -0,0 +1,42 @@
+
+      - name: Get current version
+        id: get_version
+        run: |
+          # Get the current version from the `VERSION` file.
+          current_version=$(cat VERSION)
+          echo "Current version: $current_version"
+          echo "::set-output name=current::${current_version}"
+
+      - name: Bump version
+        id: bump_version
+        run: |
```

actionlint (1.7.12)

[error] workflow command "set-output" was deprecated. use `echo "{name}={value}" >> $GITHUB_OUTPUT` instead: https://docs.github.com/en/actions/using-workflows/workflow-commands-for-github-actions

(deprecated-commands)